### ISSUE TITLE
CandidateReference: customTextBlock lengths incorrect

### ISSUE THREAD

**Issue by akelday** [bullhorn/sdk-rest#162]
The CandidateReference entity appears to have the wrong length (255) reported for customTextBlock1 to 3.

com.bullhornsdk.data.exception.RestApiException: {
  "errorMessage" : "error persisting an entity of type: CandidateReference",
  "errorMessageKey" : "errors.cannotPersistEntity",
  "errorCode" : 500,
  "errors" : [ {
    "propertyName" : "customTextBlock1",
    "maxLength" : 255,
    "severity" : "ERROR",
    "type" : "PROPERTY_VALUE_TOO_LONG"
  } ],
  "entityName" : "CandidateReference"
}

**Comment by akelday** [comment]
Is there's any progress with this issue?  We have a customer with missing data due to this problem and they obviously want an update...

**Comment by johnsully83** [comment]
Hey there,
So the SDK actually just throws a RestApiException whenever it receives a 500 (or similar) response from the BULLHORN REST APIs.  The fact we're seeing this error implies that BH is replying to your POST/PUT with

{
"errorMessage" : "error persisting an entity of type: CandidateReference",
"errorMessageKey" : "errors.cannotPersistEntity",
"errorCode" : 500,
"errors" : [ {
"propertyName" : "customTextBlock1",
"maxLength" : 255,
"severity" : "ERROR",
"type" : "PROPERTY_VALUE_TOO_LONG"
} ],
"entityName" : "CandidateReference"
}

then the SDK sees this and just throws an exception, which contains the JSON String we're seeing here.

Basically in order to fix this issue we'll want to open a ticket with BULLHORN Support, because the fix will involve a change to the REST APIs themselves as opposed to a change in the SDK.  If you can contact BH support and open up such an issue, we can absolutely ensure this gets fixed as soon as possible.
